Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/turret.yml

```diff
@@ -0,0 +1,293 @@
+    - type: BallisticAmmoProvider
+      proto: CartridgeAntiMateriel
+      capacity: 250
+    - type: Gun
+      fireRate: 1.5
+    - type: NpcFactionMember
+      factions:
+        - SimpleHostile
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponHeavyTurret
+  name:  heavy ballistic turret
+  suffix: NanoTrasen
+  description: A heavy ballistic projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: BulletMagnum
+      capacity: 500
+    - type: Gun
+      fireRate: 3
+    - type: NpcFactionMember
+      factions:
+       - NanoTrasen
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponHeavyTurretSyndicate
+  suffix: Syndicate
+  name: heavy ballistic turret
+  description: A heavy ballistic projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: BulletMagnum
+      capacity: 250
+    - type: Gun
+      fireRate: 3
+    - type: NpcFactionMember
+      factions:
+        - Syndicate
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponHeavyTurretHostile
+  suffix: Hostile
+  name: heavy ballistic turret
+  description: A heavy ballistic projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: BulletMagnum
+      capacity: 250
+    - type: Gun
+      fireRate: 3
+    - type: NpcFactionMember
+      factions:
+        - SimpleHostile
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponShotgunTurret
+  name: shottynator
+  suffix: NanoTrasen
+  description: A .50 Pellet projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: ShellShotgun
+      capacity: 500
+    - type: Gun
+      fireRate: 1
+    - type: NpcFactionMember
+      factions:
+       - NanoTrasen
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponShotgunTurretSyndicate
+  name: shottynator
+  suffix: Syndicate
+  description: A .50 Pellet projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: ShellShotgun
+      capacity: 500
+    - type: Gun
+      fireRate: 1
+    - type: NpcFactionMember
+      factions:
+       - Syndicate
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponShotgunTurretHostile
+  name: shottynator
+  suffix: Hostile
+  description: A .50 Pellet projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: ShellShotgun
+      capacity: 500
+    - type: Gun
+      fireRate: 1
+    - type: NpcFactionMember
+      factions:
+       - SimpleHostile
+
+#laser Weapons
+
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponHitscanTurret
+  name: laser turret
+  suffix: NanoTrasen
+  description: A laser based projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: BulletEnergyGunLaser
+      capacity: 500
```

Comment:
Same here. Why not use proper lasers?
```
  - type: HitscanBatteryAmmoProvider
    proto: RedLightLaser
    fireCost: 50
  - type: Battery
    maxCharge: 5000
    startingCharge: 5000
```

Reply:
It completely bricks the turret or it won't fire anything.

Reply:
I have already test this stuff before.

Reply:
https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/c861f983-03a1-425c-a644-95fab6a2308b

Reply:
when I was with magnus it wasnt working. Also there was talk about it causing lagging/crashing before.   if those are no longer the cause ill change the impact to hitscans

Reply:
which I guess why it wasn't working was because there wasn't a battery component

Reply:
they are actual lasers now